### ISSUE TITLE
Deploy to Heroku not working

### ISSUE THREAD

**Issue by jalagrange** [FlyoverWorks/storytime#152]
Hey guys, just wanted to point out that your "Deploy to Heroku" button is not working... everything seemd to be deploying as expected but then this was the output:

"There was an issue building your app. This can mean your app.json's project is not a valid Heroku application. Please ensure your app is deployable to Heroku and try again."

and the log file:

Discovering process types
       Procfile declares types -> web, worker
       Default types for Ruby  -> console, rake
-----> Compressing... done, 56.2MB
-----> Launching... failed
 !     Push failed: error creating release
 !     Please try pushing again.
 !     If the problem persists, see http://help.heroku.com/

**Comment by dvanderbeek** [comment]
Hmm…maybe try again? I just deployed using that button. It finished successfully, and the app is up and running.


—

**Comment by jalagrange** [comment]
Alright, let me try again, il get back to you.

**Comment by dvanderbeek** [comment]
Actually, hold on for a sec. I used the button on our example app's readme. The one on here is missing some info. Let me fix it quickly

**Comment by jalagrange** [comment]
It works David, thanks!

**Comment by dvanderbeek** [comment]
Sure thing. If you have problems opening the Site Settings modal, that is because there were 2 migrations missing in the example app. I just added those, so you might need to push those up to your heroku app and migrate the database if you deployed before they got added.
